wikitemplate-IPFS.md

```diff
@@ -85,56 +130,91 @@
 ## IPNS URLs
 
 - [ ] Ensure each of the following IPNS URLs load over both `Gateway` and `Local node` modes:
-    - [ ] `ipns://brantly.eth`
-    - [ ] `ipns://en.wikipedia-on-ipfs.org`
-    - [ ] `ipns://libp2p.io/`
-    - [ ] `ipns://ipfs.io`
+   - [ ] `ipns://brantly.eth`
+   - [ ] `ipns://en.wikipedia-on-ipfs.org`
+   - [ ] `ipns://libp2p.io/`
+   - [ ] `ipns://en.wikipedia-on-ipfs.org/wiki/Tokyo`
+   - [ ] `ipns://ipfs.io`
 
 ## Gateway
 
 - [ ] Verify, on a new profile, you can load `brave://settings/ipfs`, click on the `Change` button for `IPFS public gateway address`, enter `https://cloudflare-ipfs.com/` and are presented with an interstitial page after loading `ipns://brantly.eth`.  Click `Use a public gateway` and confirm you're taken to `https://cloudflare-ipfs.com/ipns/brantly.eth/#/`.  (Alternatively, use one from `https://ipfs.github.io/public-gateway-checker/`.)
```

Comment:
Can change this to 

- [ ] Verify on a new profile, you can change IPFS public gate way address to `https://cloudflare-ipfs.com/` via  `Settings -> IPFS -> Change`
- [ ] Verify visiting `ipns://brantly.eth` loads interstitial page 
- [ ] Verify selecting `Use public gate gateway` loads `https://cloudflare-ipfs.com/ipns/brantly.eth/#/`

Reply:
Thanks; done.